Doc/library/json.rst

```diff
@@ -262,6 +262,11 @@ Basic Usage
    .. versionchanged:: 3.6
       All optional parameters are now :ref:`keyword-only <keyword-only_parameter>`.
 
+   .. versionchanged:: 3.6
+      *fp* can now be a :term:`file-like object` whose ``.read()`` method
```

Comment:
Why not use just ``` :term:`binary file` ```?

Reply:
I can go with that, was mostly matching the verbiage above (`load` after all only cares that the object has a `.read()` method)